src/content/doc-sdk-java/core/create-a-new-connection.mdx

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_position: 1
+sidebar_label: Create a new connection
+title: Java | SDK | Create a new connection
+description: The SurrealDB SDK for Java enables simple and advanced querying of a remote or embedded database.
+---
+
+import Tabs from "@components/Tabs/Tabs.astro";
+import TabItem from "@components/Tabs/TabItem.astro";
+import Label from "@components/shared/Label.astro";
+
+# Create a new connection
+
+The first step towards interacting with SurrealDB is to create a new connection to a database instance. This involves initializing a new instance of the `Surreal` class and connecting it to a database endpoint. You can then switch to a specific namespace and database, and pass required authentication credentials.
+
+This guide will walk you through the process of creating a new connection to a SurrealDB instance using the Java SDK.
+
+## Related Methods
+
+<table>
+	<thead>
+		<tr>
+			<th scope="col">Method</th>
+			<th scope="col">Description</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td scope="row" data-label="Method">
+				<a href="https://surrealdb.github.io/surrealdb.java/javadoc/com/surrealdb/Surreal.html#connect(java.lang.String)">
+					<code> driver.connect(String url) </code>
+				</a>
+			</td>
+			<td scope="row" data-label="Description">Connects to a local, remote, or embedded database endpoint</td>
+		</tr>
+        <tr>
+            <td scope="row" data-label="Method">
+				<a href="https://surrealdb.github.io/surrealdb.java/javadoc/com/surrealdb/Surreal.html#close()">
+					<code> driver.close() </code>
+				</a>
+			</td>
+            <td scope="row" data-label="Description">Closes the persistent connection to the database</td>
+        </tr>
+		<tr>
+			<td scope="row" data-label="Method">
+				<a href="https://surrealdb.github.io/surrealdb.java/javadoc/com/surrealdb/Surreal.html#useNs(java.lang.String)">
+					<code> driver.useNs(String namespace) </code>
+				</a>
+			</td>
+			<td scope="row" data-label="Description">Switch to a specific namespace</td>
+		</tr>
+		<tr>
+			<td scope="row" data-label="Method">
+				<a href="https://surrealdb.github.io/surrealdb.java/javadoc/com/surrealdb/Surreal.html#useDb(java.lang.String)">
+					<code> driver.useDb(String database) </code>
+				</a>
+			</td>
+			<td scope="row" data-label="Description">Switch to a specific database</td>
+		</tr>
+	</tbody>
+</table>
+
+## Opening a connection
+
+The `.connect()` method accepts a `String` pointing to the desired local or remote instance. Since the Java SDK supports running embedded databases, you can also specify an embedded endpoint such as `memory://` and `surrealkv://`.
+
+### Supported protocols
+This is the complete list of supported connection protocols
+
+- `http://` - Plain HTTP
+- `https://` - Secure HTTP
+- `ws://` - Plain WebSocket
+- `wss://` - Secure WebSocket
+- `memory://` - In-memory database
+- `surrealkv://` - Disk-based database
```

Comment:
Should RocksDB and the rest be here too?

Reply:
I looked through the code and noticed these were the only ones enabled, hence I left out the others